django/core/management/commands/makemigrations.py

```diff
@@ -391,7 +391,7 @@ def write_migration_files(self, changes, update_previous_migration_paths=None):
                         )
                     )
                     self.log(writer.as_string())
-        run_formatters(self.written_files)
+        run_formatters(self.written_files, stderr=self.stderr)
```

Comment:
I think we may need to add minimal testing for this, to avoid this being reverted unwillingly.

Reply:
Thanks for taking a look after a pretty hasty push 😅 .

I added one test for makemigrations. If it's looking right-track to you, would you let me know if you think it should be adapted to any other commands?

Reply:
Thanks! I think the test looks good, would you an equivalent for every modified command?